debug_toolbar/panels/sql/tracking.py

```diff
@@ -31,10 +33,15 @@ class SQLQueryTriggered(Exception):
     """Thrown when template panel triggers a query"""
 
 
-def wrap_cursor(connection, panel):
+def wrap_cursor(connection):
+    # If running a Django SimpleTestCase, which isn't allowed to access the database,
+    # don't perform any monkey patching.
```

Comment:
We should include the "Why" part of the context from the other comment. It helps me understand the justification and purpose of the code.


        # Sometimes the cursor()/chunked_cursor() methods of the DatabaseWrapper
        # instance are already monkey patched before wrap_cursor() is called.  (In
        # particular, Django's SimpleTestCase monkey patches those methods for any
        # disallowed databases to raise an exception if they are accessed.)

Reply:
@tim-schilling I think you are right that expanding the comment a bit more could be helpful.  I've made an initial attempt [here](https://github.com/jazzband/django-debug-toolbar/commit/d51611be46ae6a09152881381810f559ebad8d2a).  If you think that is an improvement, let me know and I can make it a proper PR.